nc/views.py

```diff
@@ -89,6 +89,9 @@ class QueryKeyConstructor(DefaultObjectKeyConstructor):
 query_cache_key_func = QueryKeyConstructor()
 
 
+CACHE_TIMEOUT = 60 * 60 * 24
```

Comment:
Maybe we can `settings.CACHE_COUNT_TIMEOUT` here?

Reply:
Ah didn't see that, will update.